backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -134,4 +143,15 @@ public void addCorsMappings(CorsRegistry registry) {
       reg.allowedOrigins(origins.toArray(String[]::new));
     }
   }
+
+  private static final class DefaultRequiresCsrfMatcher implements RequestMatcher {
+    // USER_ACCOUNT_REQUEST is the only path that has spring sessions enabled, so it also needs
+    // CSRF support.
+    // All other paths will break if CSRF is enabled.
+    @Override
+    public boolean matches(HttpServletRequest request) {
```

Comment:
Very optional: since `RequestMatcher` only has one method, you could replace this inner class with an inline lambda or a static method reference.